app/views/quby/v1/table/_item_table_header.html.haml

```diff
@@ -4,7 +4,7 @@
   -#TODO: if a hidden option is checked it should get its own new scale header
   - cls = ['scaleheader']
   - if question.score_header == :question_description
-    %th.option{:title => raw(question.description), :class => cls, :colspan => colspan,  :class => os_cycle}
+    %th.option{:title => raw(question.description), :class => cls, :colspan => colspan}
```

Comment:
Het geeft een warning onder nieuwe rubys over een duplicate key. Volgens mij is de eerste `cls` de juiste class hier (ook hieronder wordt de option met die `cls` gerenderd).